lib/rubocop/rspec/config_formatter.rb

```diff
@@ -22,6 +22,8 @@ def dump
 
       def unified_config
         cops.each_with_object(config.dup) do |cop, unified|
+          next if cop == 'RSpec' # Skip department-level configuration
```

Comment:
I think subtracting `['RSpec']` in the `#cops` method would be a better location for the fix. And I think maybe we can get rid of the `NAMESPACES` constant too, but that’s probably for another pull request.

Reply:
Grepping using `/^(RSpec\/)/` actually also skips bare `RSpec:`.

Reply:
> get rid of the NAMESPACES constant 

Couldn't stand this. Fixed.